templates/web/pages/org_form/org_form.tt.html

```diff
@@ -68,13 +68,13 @@ <h2>[% lang("organization_members") %]</h2>
 	</div>
 	[% END %]
 
-		<form method="post" action="/cgi/org.pl" enctype="multipart/form-data" style="margin-bottom: 20px;">
+	<form method="post" action="/cgi/org.pl" enctype="multipart/form-data" style="margin-bottom: 20px;">
 		<p>[% lang('enter_email_addresses_of_users') %]</p>
-		<textarea id="email_list" name="email_list" style="height:100px;width:50vw"></textarea>
+		<textarea id="email_list" name="email_list" style="height:100px;width:50vw" oninput="toggleInviteButton()"></textarea>
```

Comment:
oninput does not exists in HTML ! Some JS framework provide it but they transform it to an event listener.

The way to do it is to use JQuery, something like:
```javascript
$(document).ready(function () {
  // observe email entry
  $("#email_list").on("input", function(event) {
    // handle event, eg. toggle Invite button
    // you can get the textarea with event.target
  });
});
```

see https://api.jquery.com/on/

Reply:
Oh okay, I'll check this out. Thanks